dotnet-desktop-guide/net/winforms/printing/snippets/how-to-print-windows-form/vb/Form1.vb

```diff
@@ -0,0 +1,25 @@
+﻿Public Class Form1
+    
+    Dim memoryImage As Bitmap
+
+    Private Sub printDocument1_PrintPage(ByVal sender As System.Object,
+       ByVal e As System.Drawing.Printing.PrintPageEventArgs) Handles _
+       PrintDocument1.PrintPage
+        e.Graphics.DrawImage(memoryImage, 0, 0)
+    End Sub
```

Comment:
I would wrap this to two lines and give the actual code line spacing around it, it will look cleaner

```suggestion
    Private Sub printDocument1_PrintPage(ByVal sender As System.Object,
       ByVal e As System.Drawing.Printing.PrintPageEventArgs) Handles PrintDocument1.PrintPage

        e.Graphics.DrawImage(memoryImage, 0, 0)

    End Sub
```